main/main.cpp

```diff
@@ -2317,8 +2317,12 @@ bool Main::start() {
 			if (editor_embed_subwindows) {
 				sml->get_root()->set_embed_subwindows_hint(true);
 			}
-		}
+			DisplayServer::get_singleton()->set_context(DisplayServer::CONTEXT_EDITOR);
+		} else
 #endif
+		{
+			DisplayServer::get_singleton()->set_context(DisplayServer::CONTEXT_ENGINE);
+		}
```

Comment:
This will set context twice for the project manager, as it's set again further down.

It's not clear to me why the previous code wasn't working?

Reply:
The set_context call that was initially in the `game_path != null` condition is removed in this commit

I think this is why this commit moves the `set_context` call outside of the condition. https://github.com/godotengine/godot/issues/52989#issuecomment-926239102

Reply:
That makes sense, but we still should move the project manager context setter here as well, inside of the `TOOLS_ENABLED` case.